src/desktop/apps/artist/components/cta.ts

```diff
@@ -22,12 +24,22 @@ const send = {
   query,
   variables: { artistID: sd.ARTIST_PAGE_CTA_ARTIST_ID },
 }
-
+export const setCookie = () => {
+  Cookies.set("artist-page-signup-dismissed", 1, { expires: 3600000 })
```

Comment:
I _think_ the `expires` is in seconds (not milliseconds), so this might suppress it for longer than expected, might be worth a double check!